bundles/org.openhab.binding.miio/src/main/resources/ESH-INF/thing/vacuumThing.xml

```diff
@@ -106,11 +108,16 @@
 		<item-type>Switch</item-type>
 		<label>Do Not Disturb</label>
 	</channel-type>
-	<channel-type id="error_code">
+	<channel-type id="error">
 		<item-type>String</item-type>
 		<label>Error</label>
 		<state readOnly="true" />
 	</channel-type>
+	<channel-type id="error_code">
+		<item-type>Number</item-type>
+		<label>Error</label>
```

Comment:
I think the label should also be Error Code to avoid duplicated label with different values

Reply:
You're right, that absolutely makes sense.